app/experimenter/nimbus-ui/src/components/TableOverview/index.tsx

```diff
@@ -3,9 +3,81 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import { AnalysisData } from "../../lib/visualization/types";
+import { useConfig } from "../../hooks";
+import { getConfig_nimbusConfig } from "../../types/getConfig";
 
-const TableOverview: React.FunctionComponent = () => {
-  return <div data-testid="table-overview">overview table</div>;
+type TableOverviewProps = {
+  experiment: getExperiment_experimentBySlug;
+  results: AnalysisData["overall"];
+};
+
+type displayConfigOptionsProps =
+  | getConfig_nimbusConfig["firefoxMinVersion"]
+  | getConfig_nimbusConfig["channels"]
+  | getConfig_nimbusConfig["targetingConfigSlug"];
+
+const TableOverview = ({ experiment }: TableOverviewProps) => {
+  const { firefoxMinVersion, channels, targetingConfigSlug } = useConfig();
+
+  return (
+    <div className="mb-5" data-testid="table-overview">
+      <table className="table text-left h5 m-0">
+        <tbody>
+          <tr>
+            <th scope="col">
+              <p className="h6 font-weight-bold">Targeting</p>
+              <p className="h6">
+                {displayConfigLabel(
+                  experiment.firefoxMinVersion,
+                  firefoxMinVersion,
+                )}
+                +
+              </p>
+              <p className="h6">
+                {experiment.channels?.length
+                  ? experiment.channels
+                      .map((expChannel) =>
+                        displayConfigLabel(expChannel, channels),
+                      )
+                      .join(", ")
+                  : ""}
+              </p>
+              <p className="h6">
+                {displayConfigLabel(
+                  experiment.targetingConfigSlug,
+                  targetingConfigSlug,
+                )}
+              </p>
+            </th>
+            <th scope="col">
+              <p className="h6 font-weight-bold">Probe Sets</p>
+              {experiment.primaryProbeSets?.length
+                ? experiment.primaryProbeSets.map((probeSet) => (
+                    <p key={probeSet?.name} className="h6">
+                      {probeSet?.name}
+                    </p>
+                  ))
+                : ""}
+            </th>
+            <th scope="col">
+              <p className="h6 font-weight-bold">Owner</p>
+              <p className="h6">{experiment.owner?.email}</p>
+            </th>
+          </tr>
+        </tbody>
+      </table>
+      <hr className="mt-0 mb-5" />
+    </div>
+  );
+};
+
+const displayConfigLabel = (
```

Comment:
Since we're using `displayConfigLabelOrNotSet` in `TableSummary`, it could be nice to extract this out to a file under `lib/` since we'll use it elsewhere, but it's not a 1:1 exactly with that function so it might make more sense to wait until we're using this in another table.

Reply:
Yes, I did get the idea from there but it's not exactly the same which is why I didn't pull it out. We could do further refactoring later.

Reply:
Sounds good to me!